app/concepts/favorite_movie/operation/add.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module FavoriteMovie::Operation
+  class Add < Trailblazer::Operation
+    step :find_movie
```

Comment:
not related to graphql:
it's better to assign model in a separate step: step :model, fail_fast: true
if it fails, it will be handled in OperationMatcher #not_found -> MatchOperationResult #not_found and the FE will get a standard not_found message

Reply:
Thank you, corrected

Reply:
Sorry, for the misinformation) I've checked the OperationMatcher code and it will use not_found if `ctx['result.model'].failure?` is true. So, `step :model` won't work here. [step Model()](https://trailblazer.to/2.0/gems/operation/2.0/api.html#model) macro should be used instead.
And `step Model()` expects params to have an `id` (not `movie_id`), so it won't work